src/phasorpy/components.py

```diff
@@ -311,3 +315,209 @@ def graphical_component_analysis(
             counts.append(numpy.asarray(component_counts))
 
     return tuple(counts)
+
+
+def n_fractions_from_phasor(
+    real: ArrayLike,
+    imag: ArrayLike,
+    /,
+    *,
+    components_real: ArrayLike | None = None,
+    components_imag: ArrayLike | None = None,
+    components_matrix: NDArray[Any] | None = None,
```

Comment:
I would prefer singular names (`component_real`, not `components_real`) and not needing to pass `component_real` and `component_imag` by name, for example:

```suggestion
    real: ArrayLike,
    imag: ArrayLike,
    /,
    component_real: ArrayLike | None = None,
    component_imag: ArrayLike | None = None,
    *,
    component_matrix: ArrayLike | None = None,
```

Reply:
Never mind the singular names. I see that the other functions in the components module also use these names. It's more important to be consistent.

Reply:
Hi @cgohlke, thanks for pointing this out, I also think is better to set this parameters as positional. Regarding the names, I also like the singular form, but since the components will always be more than one (since it doesn't make sense a one component analysis), I think it's more intuitive to leave it as plural. But if you think is better singular I can also make the modifications in the other functions of the module to be consistent with this one.

Reply:
Sorry, I'm used to "component matrix", e.g. from PCA.